docs/operations/database-backup.md

```diff
@@ -8,7 +8,7 @@ There are 2 main ways to perform backups:
 
 ### Disk Snapshotting 
 
-If you the infrastructure is virtualized, disk _snapshots_ is an option and the easiest to perform backup and restore operations.
+If your infrastructure is virtualized, disk _snapshots_ is an option and the easiest to perform backup and restore operations.
```

Comment:
You could mention ZFS (and perhaps BTRFS) as the superior way to keep your data safe and take snapshots that you can ship to different servers and media.

Reply:
We should add this to a dedicated section on ZFS.